DeployJenkinsfile

```diff
@@ -18,6 +18,8 @@ pipeline {
         choice(name: 'DEPLOYMENT_TYPE', choices:'cf3-release-candidate\ncf3-staging\ncf3-integration\nvpc\neu-central\ngovw\nperf-cf3', description: 'This specifies which point of presence to deploy to')
         booleanParam(name: 'MAP_ROUTES_TO_DEPLOYED_APP', defaultValue: true, description: 'Determines if the published routes are mapped to the newly deployed app. Defaults to true.')
         booleanParam(name: 'RUN_FULL_ACCEPTANCE_TESTS', defaultValue: true, description: 'Determines whether to run the full acceptance test suite against the freshly deployed UAA. If left unchecked, it will run at least the basic acceptance tests.')
+        string(name: 'UAA_CF_RELEASE_BRANCH', defaultValue: 'master',
```

Comment:
I think we should still call this `UAA_CI_CONFIG_BRANCH` as a descriptive name. Even though the repo is called cf-release, its purpose is to store "CI configuration". You can put the actual branch name in the "description" so there's no confusion.

Reply:
Sure

Reply:
Actually I think we should call it what it is, a branch in the uaa-cf-release repo, but I don't have the energy to debate it :D